<launch>
    <param name="param_file" type="str" value="$(find weeding_machine)/cfg/param_offline.yaml" />

    <!--   rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find weeding_machine)/rviz/main_2.rviz"/>

    <!--     tf_manager -->
    <node pkg="weeding_machine" type="tf_manager.py" name="tf_manager" output="screen"/>

    <!--     lane_det -->
    <node pkg="weeding_machine" type="lane_det_gaus.py" name="lane_det_gaus" output="screen"/>

    <node pkg="weeding_machine" type="pyqt_show.py" name="ros_gui" output="screen" > -->
        <param name="image_topic1"       value="/front_rgb_cam/image_raw" />
    </node>

<!--     <node pkg="rosbag" type="play" name="play_file1" output="screen" args="-r 0.2 /home/zhong/my_workspace/src/bag/9.bag"> -->
<!--     <param name="start_paused" value="false"/> -->
<!--     </node> -->
<!--     <node pkg="rosbag" type="play" name="play_file2" output="screen" args="-r 0.2 /home/zhong/my_workspace/src/bag/10.bag"> -->
<!--     <param name="start_paused" value="false"/> -->
<!--     </node> -->
    <!--     weeder_cmd_serial_comm -->
    <node pkg="weeding_machine" type="weeder_cmd_serial_comm.py" name="weeder_cmd_serial_comm" output="screen"/>

        logger
    <node pkg="weeding_machine" type="logger.py" name="logger" output="screen"/>
</launch>
